client/src/components/TagBadge/TagBadge.component.tsx

```diff
@@ -0,0 +1,50 @@
+import { Link } from 'react-router-dom'
+import { getRedirectURL } from '../../util/urlparser'
+import './TagBadge.styles.scss'
+import { useGoogleAnalytics } from '../../contexts/googleAnalytics'
+import * as FullStory from '@fullstory/browser'
+import { TagType } from '~shared/types/base'
+import { Agency } from '../../services/AgencyService'
+
+const TagBadge = ({
+  tagName,
+  tagType,
+  agency,
+}: {
+  tagName: string
+  tagType: TagType
+  agency: Agency
+}): JSX.Element => {
+  const TagComponent = <div className="tag-badge">{tagName.toUpperCase()}</div>
+
+  const googleAnalytics = useGoogleAnalytics()
+
+  const sendClickTagEventToAnalytics = () => {
+    const timeToTagClick = new Date().getTime() - googleAnalytics.appLoadTime
```

Comment:
no need to change but just FYI these are equivalent
```suggestion
    const timeToTagClick = Date.now() - googleAnalytics.appLoadTime
```

Reply:
Nice, looks cleaner, thanks! Will change it together with other places where this is done.